src/gui/MItem_network.cpp

```diff
@@ -177,6 +177,11 @@ MI_NET_IP::MI_NET_IP(NetDeviceID device_id)
     , device_id(device_id) //
 {
     index = netdev_get_ip_obtained_type(this->device_id());
+    if (index == NETDEV_STATIC) {
+        this->SetIndex(1);
```

Comment:
You're using 5 lines of code for something that could be solved by a ternary operator. Or even no ternary operator in this case possibly, bool implicitly casts to int.